Backend/api/validator.py

```diff
@@ -25,19 +26,19 @@ def send_confirmation_mail(receiver_email):
         f"Alumni Portal Team"
     )
 
-    smtp = smtplib.SMTP(smtp_server, port=port)
-    smtp.ehlo()  # send the extended hello to our server
-    smtp.starttls()  # tell server we want to communicate with TLS encryption
-    smtp.login(sender_email, password)  # login to our email server
-    try:
-        smtp.sendmail(sender_email, receiver_email,
-                      message)
-        smtp.quit()
-    except: # pylint: disable=bare-except
-        print("FAILED TO SEND EMAIL")
+    # smtp = smtplib.SMTP(smtp_server, port=port)
```

Comment:
You should put a comment here why did you remove this code (we know why but for better documentation

Reply:
I am planning to use something completely different. However, if university infrastructure gets stable we can put it back

Reply:
I agree but I am just about writing why did you do it, so that if someone or you read the code, you can know why it was done this way